### ISSUE TITLE
SCSS errors not showing in console

### ISSUE THREAD

**Issue by DaveOrDead** [adrianhall/stylelint-loader#2]
This could be something I've done wrong.

I was following this tutorial (https://shellmonger.com/2016/01/29/integrating-sass-lint-with-webpack/comment-page-1/) to figure out how to add stylelint to Webpack.

I’m running Eslint for my js which is working fine. But for some reason nothing happens with my scss no matter how badly I break the rules I set.

I’m using the following in my webpack config (Full config file here: http://codepen.io/davislurve/pen/gPdxzB):

```
stylelint: {
    configFile: path.join(__dirname, ‘./.stylelint.config.js’),
    rules: {
        ‘function-url-quotes’: ‘single’,
        ‘indentation’: [ 4, { warn: true } ],
        ‘string-quotes’: ‘single’
    }
}
```

but when I use double quotes or 2 space indentation in my SCSS code nothing appears in the console.

It also doesn’t throw an error if I change the config file name e.g:

`configFile: path.join(__dirname, ‘./SOME_RANDOM_FILE.config.js’),`

I was expecting an error saying the file can’t be found or something. So I’m wondering if it’s even being initiated properly.

**Comment by adrianhall** [comment]
Can you create a repository on github for your failing project, so I can clone it and try it out?

**Comment by adrianhall** [comment]
Ref: https://github.com/adrianhall/stylelint-loader/commit/11a27364fee5eb3e06d70e21e8dc4cc9c6ed754c - check-in to warn when the config file does not exist.

**Comment by DaveOrDead** [comment]
Thanks so much for doing this: https://github.com/davislurve/stylelint_test

**Comment by adrianhall** [comment]
Well, I know WHY you are not seeing any output.  The .stylelint.config.js file is wrong.  See the reference here:  https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules.json

Basically, you need to alter your file like this snippet:

```
        // Color
        "color-hex-case": "lower",
        "color-hex-length": "short",
        "color-no-invalid-hex": true,
```

There is a bug here - actually two, one of which I've fixed already.  The "Invalid rule" is not reported.  I'm going to fix that and re-release.

Thanks for bringing this to my attention.  I'll close this issue once the new version is released.

**Comment by adrianhall** [comment]
Fixed in v0.2.2 of stylelint-loader
